dask_ml/model_selection/methods.py

```diff
@@ -549,7 +519,6 @@ def create_cv_results(
                 n_splits,
                 n_candidates,
                 splits=True,
-                weights=weights
```

Comment:
any reason why `weights` parameter is removed?

Reply:
In the way it works for the train folds the weights are passed around in the `fit_params` object which is then extracted with the CV object that has the cache. This CV object will get the train weights from `fit_params` and with this patch it also get the test weights too.

TLDR; fit_params moves around the training process so the weights are fetched from that object instead of passing around the weights explicitly

Reply:
good idea!